### PR TITLE
burpsuite: add page

### PR DESCRIPTION
**Pull request by TerribleCodes**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented: 2021.10.3-10704**
#6863

### CODE REVIEW

File: pages/linux/burpsuite.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# burpsuite
+
+> A GUI based application that's used in web application penetration testing.
+> CLI is used to control the startup behavior of Burpsuite application.
```
Comment (pixelcmtd) [inline pages/linux/burpsuite.md:4]: What else should it do?

```suggestion
```

File: pages/linux/burpsuite.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# burpsuite
+
+> A GUI based application that's used in web application penetration testing.
```
Comment (pixelcmtd) [inline pages/linux/burpsuite.md:3]:
```suggestion
> A GUI based application mainly used in web application penetration testing.
```

File: pages/linux/burpsuite.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+> CLI is used to control the startup behavior of Burpsuite application.
+> More information: <https://portswigger.net/burp/documentation/desktop/getting-started/launch-from-command-line>.
+
+- Start burpsuite application:
```
Comment (pixelcmtd) [inline pages/linux/burpsuite.md:7]:
```suggestion
- Start Burp Suite:
```

File: pages/linux/burpsuite.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+`burpsuite`
+
+- Start burpsuite with default configurations:
```
Comment (pixelcmtd) [inline pages/linux/burpsuite.md:11]:
```suggestion
- Start Burp Suite with defaults, overriding the user's configuration:
```

File: pages/linux/burpsuite.md (hunk 1 above)
Comment (TerribleCodes) [reply pages/linux/burpsuite.md]: @pixelcmtd 

Well, the CLI is only used to control startup behavior. So I don't think it's necessary to add more here. So I committed the suggested change there.

File: pages/linux/burpsuite.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open a specific project file:
+
+`burpsuite --project-file={{path/to/file}}`
+
+- Load a specific configuration file:
+
+`burpsuite --config-file={{path/to/file}}`
+
+- Prevent loading extensions on startup:
```
Comment (navarroaxel) [inline pages/linux/burpsuite.md:22]:
```suggestion
- Start without extensions:
```

File: pages/linux/burpsuite.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# burpsuite
+
+> A GUI based application mainly used in web application penetration testing.
+> More information: <https://portswigger.net/burp/documentation/desktop/getting-started/launch-from-command-line>.
+
+- Start Burp Suite:
+
+`burpsuite`
+
+- Start Burp Suite with defaults, overriding the user's configuration:
```
Comment (navarroaxel) [inline pages/linux/burpsuite.md:10]: Perhaps this better
```suggestion
- Start Burp Suite using the default configuration:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**